bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -97,14 +97,14 @@ spec:
             command:
             - /usr/share/openvswitch/scripts/ovs-ctl
             - status
-          initialDelaySeconds: 15
+          initialDelaySeconds: 45
```

Comment:
Is this a regression from a change in OVS?

Reply:
It's unclear to me. but I think we saw this during a 4.4 to 4.5 upgrade, and it may be a function of how loaded the node is vs. when the ovs pods come up the first time. I think we saw this most in a scale setup.